language-support/ts/codegen/tests/src/DA/Test/Daml2js.hs

```diff
@@ -176,20 +171,20 @@ tests yarn damlc daml2js davl = testGroup "daml2js tests"
           , "A" </> "B" </> "D" </> "index"
           , "A" </> "B" </> "D" </> "module"
           ]
-        assertFileDoesNotExist (projectTs </> "src" </> "A" </> "B" </> "module.ts")
+        assertFileDoesNotExist (projectTs </> "lib" </> "A" </> "B" </> "module.js")
 
-        withCurrentDirectory (projectTs </> "src") $ do
+        withCurrentDirectory (projectTs </> "lib") $ do
           let reexportIndex name =
                 [ "import * as " <> name <> " from './" <> name <> "';"
-                , "export import " <> name <> " = " <> name <> ";"
+                , "export { " <> name <> " } ;"
                 ]
           let reexportModule = ["export * from './module';"]
-          indexContents <- T.lines <$> T.readFileUtf8 "index.ts"
+          indexContents <- T.lines <$> T.readFileUtf8 "index.d.ts"
           assertBool "index.ts does not reexport A" (reexportIndex "A" `isPrefixOf` indexContents)
-          assertFileLines ("A" </> "index.ts") (reexportIndex "B" ++ reexportModule)
-          assertFileLines ("A" </> "B" </> "index.ts") (reexportIndex "C" ++ reexportIndex "D")
-          assertFileLines ("A" </> "B" </> "C" </> "index.ts") reexportModule
-          assertFileLines ("A" </> "B" </> "D" </> "index.ts") reexportModule
+          assertFileLines ("A" </> "index.d.ts") (reexportIndex "B" ++ reexportModule)
+          assertFileLines ("A" </> "B" </> "index.d.ts") (reexportIndex "C" ++ reexportIndex "D")
+          assertFileLines ("A" </> "B" </> "C" </> "index.d.ts") reexportModule
+          assertFileLines ("A" </> "B" </> "D" </> "index.d.ts") reexportModule
```

Comment:
Would it make sense to add assertions for the `.js` files here as well?

Reply:
The way I understood this code it was more like a lazily written unit test (since it piggybacks on top of the integration test) to check that the logic for generating the index tree is correct. Therefore it didn’t really seem useful to test that the rendering logic to commonjs is correct and the commonjs module stuff looks horrible so I’d rather test the logic using the nice ES6 modules.